src/include/mom_func.h

```diff
@@ -235,6 +235,16 @@ extern int  is_comm_up(int);
 #define PE_PROLOGUE	1
 #define PE_EPILOGUE	2
 
+typedef enum {
+	PRE_FINISH_SUCCESS,
+	PRE_FINISH_SUCCESS_JOB_SETUP_SEND,
+	PRE_FINISH_FAIL,
+	PRE_FINISH_FAIL_JOB_SETUP_SEND,
+	PRE_FINISH_FAIL_JOIN_EXTRA,
+	PRE_FINISH_FAIL_NEW_CPUSET,
+	PRE_FINISH_FAIL_VERIFY_BGL
```

Comment:
We have agreed the BGL is one of the platforms that will be removed as part of https://pbspro.atlassian.net/browse/PP-1206

If you leave it in, it will be removed when PP-1206 is addressed.

Reply:
Ok. For now, I'll just remove it from the code that I've introduced.